assets/sass/_reset.scss

```diff
@@ -58,24 +58,8 @@ button {
   box-sizing: border-box;
 }
 
-// Viewport resizing
```

Comment:
Parcel marked these as deprecated and errored during css compilation. Seems like this isn't really a thing: https://caniuse.com/?search=%40viewport

Reply:
We already have `    <meta name="viewport" content="width=device-width, initial-scale=1.0" />` in the `baseof.html` template which is the more accepted way to do this.

Reply:
👍